File: taller/SergioDeIscar-taller-java/SergioDeIscar-taller-java/src/main/java/org/example/models/Chapista.java
package org.example.models;

public final class Chapista extends Trabajador{
    public Chapista(String nombre, int experiencia, int horasDiarias) {
        super(nombre, experiencia, horasDiarias);
    }

    int salario = 1800;

    @Override
    public void comer() {
        System.out.println("El trabajador chapista " + nombre + " come.");
    }

    @Override
    public int getSalario() {
        return salario;
    }

    public void arreglarChapa(){
        System.out.println("El chapista " + nombre + " esta arreglando la chapa.");
    }
}

File: taller/SergioDeIScar-taller-kotlin/SergioDeIscar-taller-kotlin/src/main/kotlin/models/Chapista.kt
package taller.models

class Chapista(nombre: String, experiencia: Int, horasDiarias: Int): Trabajador(nombre, experiencia, horasDiarias) {
    override val salario: Int = 1700
    override fun comer() {
        println("El trabajador chapista $nombre come.")
    }

    fun arreglarChapa(){
        println("El chapista $nombre esta arreglando la chapa.")
    }
}